src/jit/liveness.cpp

```diff
@@ -1400,6 +1400,54 @@ bool Compiler::fgMarkIntf(VARSET_VALARG_TP varSet1, VARSET_VALARG_TP varSet2)
 #endif
 }
 
+bool Compiler::fgMarkIntf(VARSET_VALARG_TP varSet, unsigned varIndex)
```

Comment:
Can you add a method header comment for this new method
using  the the one above (lines 1344-1353) as a template